mods/creative/inventory.lua

```diff
@@ -8,60 +8,33 @@ function creative.init_creative_inventory(player)
 		start_i = 0
 	}
 
-	minetest.create_detached_inventory("creative_" .. player_name, {
-		allow_move = function(inv, from_list, from_index, to_list, to_index, count, player2)
-			if not to_list == "main" then
-				return count
-			else
-				return 0
-			end
-		end,
-		allow_put = function(inv, listname, index, stack, player2)
-			return 0
-		end,
-		allow_take = function(inv, listname, index, stack, player2)
-			return -1
-		end,
-		on_move = function(inv, from_list, from_index, to_list, to_index, count, player2)
-		end,
-		on_put = function(inv, listname, index, stack, player2)
-		end,
-		on_take = function(inv, listname, index, stack, player2)
-			if stack and stack:get_count() > 0 then
-				minetest.log("action", player_name .. " takes " .. stack:get_name().. " from creative inventory")
-			end
-		end,
-	}, player_name)
-
 	return player_inventory[player_name]
 end
 
 function creative.update_creative_inventory(player_name, tab_content)
 	local creative_list = {}
 	local inv = player_inventory[player_name] or
-			creative.init_creative_inventory(minetest.get_player_by_name(player_name))
-	local player_inv = minetest.get_inventory({type = "detached", name = "creative_" .. player_name})
+		creative.init_creative_inventory(minetest.get_player_by_name(player_name))
 
 	for name, def in pairs(tab_content) do
 		if not (def.groups.not_in_creative_inventory == 1) and
-				def.description and def.description ~= "" and
-				(def.name:find(inv.filter, 1, true) or
-					def.description:lower():find(inv.filter, 1, true)) then
+		   def.description and def.description ~= ""	   and
+		  (def.name:find(inv.filter, 1, true)		   or
+		   def.description:lower():find(inv.filter, 1, true)) then
```

Comment:
Two tabs for additional `if` lines, I think you may use spaces afterwards to line up.